meson_options.txt

```diff
@@ -0,0 +1,11 @@
+# Feature options
+option('enable-tests', type : 'boolean', value : true, description : 'Enable tests')
+option('enable-check', type : 'boolean', value : true, description : 'Enable check tests, need the enable-tests flag set')
+option('enable-graphviz', type : 'boolean', value : false, description : 'Enable graphviz usage')
```

Comment:
What graphviz does? Isn't it better to describe something like: `enable-plotting`?

Somehow, I feel tied to the implementation

Reply:
Graphviz allows us to create a plot of the pipeline, like here: https://developer.ridgerun.com/wiki/index.php/GstShark_-_Graphic_tracer

I changed this to your suggestion, but underneath, the implementation will keep using graphviz.